package.json

```diff
@@ -71,7 +71,8 @@
     "typescript-eslint": "^8.27.0",
     "vite-plugin-node-polyfills": "^0.23.0",
     "vitest": "^3.1.2",
-    "yaml": "~2.7.0"
+    "yaml": "~2.7.0",
+    "es-toolkit": "~1.36.0"
```

Comment:
I would prefer if we didn't need to add another dependency just for this small use case.
It should also be added to the `packages/openapi3/package.json` here this is just for our workspace dev dependency which have no effect in published packages